### ISSUE TITLE
Curator issue when annotating a phrase with space in the beginning 

### ISSUE THREAD

**Issue by danyaljj** [IllinoisCogComp/illinois-cogcomp-nlp#56]
I was trying to re-run `CreateTestTAResource` which uses curator to annotate a bunch of sentences. 
There is this sentence which has space at the beginning: 

https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java#L86-L87 

when I remove the space, it runs fine; otherwise it throws the following exception: 
```
text =   We have three kinds of pastries. One of them has chocolate and the other 2 don't. 
java.lang.ArrayIndexOutOfBoundsException: -1
	at edu.illinois.cs.cogcomp.core.datastructures.textannotation.AbstractTextAnnotation.getToken(AbstractTextAnnotation.java:186)
	at edu.illinois.cs.cogcomp.curator.CuratorDataStructureInterface.makeDependencyTree(CuratorDataStructureInterface.java:559)
	at edu.illinois.cs.cogcomp.curator.CuratorDataStructureInterface.alignForestToDependencyView(CuratorDataStructureInterface.java:502)
	at edu.illinois.cs.cogcomp.curator.CuratorClient.getTextAnnotationView(CuratorClient.java:248)
	at edu.illinois.cs.cogcomp.curator.CuratorAnnotator.addView(CuratorAnnotator.java:46)
	at edu.illinois.cs.cogcomp.annotation.Annotator.getView(Annotator.java:68)
	at edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation.addView(TextAnnotation.java:109)
	at edu.illinois.cs.cogcomp.curator.CuratorAnnotatorService.addView(CuratorAnnotatorService.java:199)
	at edu.illinois.cs.cogcomp.edison.utilities.CreateTestTAResource.getTextAnnotations(CreateTestTAResource.java:249)
	at edu.illinois.cs.cogcomp.edison.utilities.CreateTestTAResource.<init>(CreateTestTAResource.java:53)
	at edu.illinois.cs.cogcomp.edison.utilities.CreateTestTAResource.main(CreateTestTAResource.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
```

Specifically it throw exception when it tries to add the DEPENDENCY view: 
```
annotator.addView(ta, ViewNames.DEPENDENCY);
```

fyi @christos-c @bhargav

**Comment by mssammon** [comment]
I think the problem is here:
```
            nonDefaultValues.put(CuratorConfigurator.RESPECT_TOKENIZATION.key,
                    CuratorConfigurator.TRUE);
```
-- this makes no sense given the actual strings being used as source text, which are not "whitespace-tokenized".  Fix by setting the second argument to `CuratorConfigurator.FALSE`.

**Comment by christos-c** [comment]
Fixed in #62
